packages/website/src/widgets/citation/ui/search/comparative-search.tsx

```diff
@@ -42,48 +46,75 @@ export default function ComparativeSearch(props: ComparativeSearchProps) {
     );
   }
 
-  return (
-    <>
-      <CitationExplorerTitle onClose={onClose}>Compare Mode</CitationExplorerTitle>
+  useEffect(() => {
+    if (regionType) {
```

Comment:
I removed the useEffect hook and turned it into a function that returns a currentStep value. Thought about using a useCallback or useMemo, but I don't think there's a benefit to it since it's not a costly calculation. Lmk what you guys think though!